app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/topBar/userMenu.tsx

```diff
@@ -0,0 +1,81 @@
+/** @file The UserMenu component provides a dropdown menu of user actions and settings. */
+import React, { PropsWithChildren, useState } from 'react'
+import * as auth from '../../../authentication/providers/auth'
+import ChangePasswordModal from './changePasswordModal'
+
+// ================
+// === UserMenu ===
+// ================
+
+/** This is the UI component for a UserMenu list item.
+ * The main interaction logic is in the onClick injected by UserMenu. */
```

Comment:
`onClick` and `UserMenu` should be in backticks (`` ` ``).
(actually, `UserMenu` should probably be `{@link UserMenu}`, but i'd say to do that only if the link properly works)